gulpfile.js

```diff
@@ -26,6 +26,10 @@ const scriptFiles = [
   path.join(scriptsDir, 'video.js')
 ]
 
+buildScssString = (filename) => {
+    return '@import "' + filename.slice(1, (filename.length-5)) + '";\n';
```

Comment:
instead of using that magic 5 I'd better used '.scss'.lenght ('.scss' also can be a constant)